net-libs/libtorrent/libtorrent-0.13.6.recipe

```diff
@@ -0,0 +1,79 @@
+SUMMARY="BitTorrent library written in C++ for *nix"
+DESCRIPTION="libtorrent is a C++ implementation of the BitTorrent protocol \
+with the goals of being efficient and easy to use."
+HOMEPAGE="http://libtorrent.rakshasa.no/"
+COPYRIGHT="2005-2016 Jari Sundell"
+LICENSE="GNU LGPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/rakshasa/libtorrent/archive/$portVersion.zip"
+CHECKSUM_SHA256="6841043785e0800cc6a7d8f9a1d213495946e3103c2557aaba63fcd78d7b2597"
+
+ARCHITECTURES="x86 ?x86_gcc2 ?x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	libtorrent$secondaryArchSuffix = $portVersion
+	lib:libtorrent$secondaryArchSuffix = $portVersion compat >= 0.13
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	lib:libcrypto$secondaryArchSuffix
+	cmd:gettext$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libtorrent${secondaryArchSuffix}_devel = $portVersion
+	devel:libtorrent$secondaryArchSuffix = $portVersion compat >= 0.13
+	"
+REQUIRES_devel="
+	haiku${secondaryArchSuffix}_devel
+	libtorrent$secondaryArchSuffix == $portVersion base
+	cmd:pkg_config$secondaryArchSuffix_devel
+	cmd:gettext$secondaryArchSuffix_devel
+	devel:libcppunit$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libcppunit$secondaryArchSuffix
+	devel:libcrypto$secondaryArchSuffix
+	devel:libgettextlib$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:libtoolize
```

Comment:
If I'm not mistaken, using **`cmd:libtoolize$secondaryArchSuffix`** (instead of just **`cmd:libtoolize`**) is recommended or required.
(I've learned that with @korli.)